raiden_installer/web.py

```diff
@@ -375,7 +379,7 @@ def _run_swap(self, **kw):
                     redirect_url = self.reverse_url("launch", configuration_file.file_name)
                     next_page = "You are ready to launch Raiden! ..."
 
-                self._send_summary(["Congratulations! Swap Successful!", next_page])
+                self._send_summary(["Congratulations! Swap Successful!", next_page], icon=token_ticker)
```

Comment:
Should we handle the Error Case as well with an icon (Line 393) ? One suggestion: Lets introduce a boolean `successful` which indicates if the Transaction went through. In the Error case we could add like a "splash" symbol indicating better that something went wrong

Reply:
And also, do you want to show the swapped token or the upcoming one?

Reply:
I want to show the swapped token.

Reply:
We could also just set the icon variable to `error` in the error case and add such an image. That way we would not need to have another argument that needs to be checked.

Reply:
also good idea

Reply:
lets do it this way